Packs/AzureDevOps/Integrations/AzureDevOps/AzureDevOps.py

```diff
@@ -1639,16 +2509,16 @@ def main() -> None:
             return_results(user_remove_command(client, args))
 
         elif command == 'azure-devops-pull-request-create':
-            return_results(pull_request_create_command(client, args))
+            return_results(pull_request_create_command(client, args, params.get('repository'), params.get('project')))
```

Comment:
instead of calling `.get('repository')` over and over again (lines 2518, 2521), assign it once above please. Same for `project`